source/Government.cpp

```diff
@@ -231,6 +252,8 @@ void Government::Load(const DataNode &node)
 			bribe = add ? bribe + child.Value(valueIndex) : child.Value(valueIndex);
 		else if(key == "fine")
 			fine = add ? fine + child.Value(valueIndex) : child.Value(valueIndex);
+		else if(child.Token(0) == "bribe factor" && child.Size() >= 2)
```

Comment:
Use key instead of child.token(0)
Also child.size >=2 is the same as hasvalue I think, so no need to check it (line243)
```suggestion
		else if(key == "bribe factor")
```